src/backend/distributed/commands/distribute_object_ops.c

```diff
@@ -374,6 +374,15 @@ static DistributeObjectOps Any_Rename = {
 	.address = NULL,
 	.markDistributed = false,
 };
+static DistributeObjectOps Any_SecLabel = {
+	.deparse = DeparseSecLabelStmt,
+	.qualify = NULL,
+	.preprocess = PreprocessSecLabelStmt,
+	.postprocess = PostprocessSecLabelStmt,
+	.operationType = DIST_OPS_ALTER,
+	.address = SecLabelStmtObjectAddress,
+	.markDistributed = false,
```

Comment:
Shouldn't this be true? Also, can we add tests to check pg_dist_object?

Reply:
`SecLabelStmt` does not need to mark anything as distributed, so it's false. Consider it similar to `GrantRoleStmt` (`Any_GrantRole`).